cirq-core/cirq/ops/gateset.py

```diff
@@ -422,3 +424,22 @@ def __str__(self) -> str:
         if self.name:
             header += self.name
         return f'{header}\n' + self._gates_str_str
+
+    def _json_dict_(self) -> Dict[str, Any]:
+        return {
+            'gates': list(self.gates),
```

Comment:
This needs to have a deterministic order for consistent serialization.

Reply:
done